packages/app-js/src/index.css

```diff
@@ -21,6 +21,12 @@
   }
 }
 
+.js--Buttons {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 0;
```

Comment:
Suggest we align there right, it looks sightly better. (Not hung-up either way about it)

Reply:
I agree, but that doesn't work with the CSS resize solution.
If the buttons are placed inside the resizable container, they can't be positioned absolute outside the container because of the overflow setting. They just disappear.

Reply:
What I could do is just add a padding-top inside the editor...

Reply:
<img width="1132" alt="Screenshot 2019-05-10 at 17 53 22" src="https://user-images.githubusercontent.com/125398/57540233-87872300-734c-11e9-8fb2-ea591a2b2a82.png">

Reply:
Like the second one, as weird as it sounds...